stories/demos/HeaderFilters.tsx

```diff
@@ -84,101 +99,154 @@ export function HeaderFilters() {
-            <select
-              className={filterClassname}
-              value={p.value}
-              onChange={(e) => p.onChange(e.target.value)}
-            >
-              <option value="All">All</option>
-              <option value="Bug">Bug</option>
-              <option value="Improvement">Improvement</option>
-              <option value="Epic">Epic</option>
-              <option value="Story">Story</option>
-            </select>
-          </div>
-        )
+        headerCellClass,
+        headerRenderer: enableFilterRow
+          ? (p) => (
+              <FilterRenderer {...p}>
+                {(filters) => (
+                  <select
+                    className={filterClassname}
+                    value={filters.issueType}
+                    onChange={(e) =>
+                      setFilters({
+                        ...filters,
+                        issueType: e.target.value
+                      })
+                    }
+                  >
+                    <option value="All">All</option>
+                    <option value="Bug">Bug</option>
+                    <option value="Improvement">Improvement</option>
+                    <option value="Epic">Epic</option>
+                    <option value="Story">Story</option>
+                  </select>
+                )}
+              </FilterRenderer>
+            )
+          : undefined
       },
       {
         key: 'developer',
         name: 'Developer',
-        filterRenderer: (p) => (
-          <div className={filterContainerClassname}>
-            <Select
-              value={p.value}
-              onChange={p.onChange}
-              options={developerOptions}
-              styles={{
-                option: (provided) => ({
-                  ...provided,
-                  padding: 10
-                }),
-                control: (provided) => ({
-                  ...provided,
-                  height: 30,
-                  minHeight: 30,
-                  padding: 0,
-                  lineHeight: 'normal'
-                }),
-                container: (provided) => ({
-                  ...provided,
-                  width: '100%'
-                })
-              }}
-              menuPortalTarget={document.body}
-            />
-          </div>
-        )
+        headerCellClass,
+        headerRenderer: enableFilterRow
+          ? (p) => (
+              <FilterRenderer {...p}>
+                {(filters) => (
+                  <>
+                    <input
+                      className={filterClassname}
+                      value={filters.developer}
+                      onChange={(e) =>
+                        setFilters({
+                          ...filters,
+                          developer: e.target.value
+                        })
+                      }
+                      list="developers"
+                    />
+                    <datalist id="developers">
```

Comment:
Alternatively it doesn't have to be rendered here, you can render it as a sibling to the grid.

Reply:
👍